src/github.com/stellar/federation/request_handler.go

```diff
@@ -65,23 +63,7 @@ func (rh *RequestHandler) MakeDatabaseRequest(query string, w http.ResponseWrite
 
 	log.Print("Federation record found")
 
-	var usernameBuffer bytes.Buffer
-	usernameBuffer.WriteString(record.Username)
-	usernameBuffer.WriteString("*")
-	usernameBuffer.WriteString(rh.config.Domain)
-
-	record.Username = usernameBuffer.String()
```

Comment:
Removing code above breaks reverse federation. Probably none of the organizations will add a separate column in their DB to hold full stellar address so we need to concatenate it with a domain name.